tests/main/interfaces-upower-observe/test-snapd-upower/snapcraft.yaml

```diff
@@ -2,7 +2,7 @@ name: test-snapd-upower
 base: core
 version: 0.99.4-3
 summary: snapd upower test snap
-description: |
+details: |
```

Comment:
this shouldn't be changed, this is not a spread test

Reply:
This is correct now.